docs/development/release.md

```diff
@@ -57,26 +71,42 @@ Note: This is for 1.10.3 or later.
 
 ### 6. Create a pull request against the master branch
 
-Create a PR against the `master` branch. Verify that the Travis CI build for the PR succeeds and then merge to master. Once merged, the release branch can be deleted.
+Create a PR against the feature branch. Verify that the Bamboo build for the PR succeeds and then merge to the feature branch. Once merged, the release branch can be deleted.
 
 ### 7. Create a git tag for the release
-
-Publishing of new releases is handled by Travis CI and is triggered when the release tag is pushed to Github. This tag should be in the format `v1.2.3`, where `1.2.3` is the new version.
-
 Create and push a new git tag:
 
 ```bash
   $ git tag -a v1.x.x -m "Release 1.x.x"
   $ git push origin v1.x.x
 ```
+### 8. Running the deployment
+
+Publishing of new releases is handled by a Bamboo release plan and is manually triggered.
+
+If you created a new release plan in step one, you will need to create a new bamboo deployment plan
+
+#### Creating a Bamboo Deployment plan:
+
+* In the Cumulus Core project (https://ci.earthdata.nasa.gov/browse/CUM-CBA), click Actions -> Configure Plan
+
+* Scroll to the bottom of the branch list and click `Create Plan Branch`
+
+* Click `Create plan branch manually`
+
+* Add the values in that list.   Choose a display name that makes it *very* clear this is a deployment branch plan.    `Release (branch name)` seems to work well.    *Make sure* you select the correct branch
+
+* **Important** Deselect Enable Branch - if you do not do this, it will immediately fire off a build
+
+* **Immediately** go to plan configuration on the `Branch Details` tab, and enable `Change trigger`.  Set the `Trigger type` to manual, this will prevent commits to the branch from triggering the build plan
 
-Travis will build and run tests against that tagged release, publish the new packages to NPM, and then run the integration tests using those newly released packages.
+* Go to the branch plan and set GIT_PR, USE_NPM_PACKAGES and PUBLISH_FLAG to true.  Select a DEPLOYMENT appropriate for the release (otherwise it will use the default last committer)
```

Comment:
DEPLOYMENT should be the `cumulus-from-npm`

Reply:
@laurenfrederick  I think that  works fine unless we're maintaining an incompatible backport branch.   e.g. if 1.13.x doesn't roll forward/back from 1.14.x.     If we want to manage that manually or delete stacks or something that seems fine.

Reply:
```suggestion
* Go to the branch plan and set GIT_PR, USE_NPM_PACKAGES and PUBLISH_FLAG to true.  Select a DEPLOYMENT appropriate for the release (otherwise it will use the default last committer)
* Go to the branch plan and set GIT_PR, USE_NPM_PACKAGES and PUBLISH_FLAG to true.  Select a DEPLOYMENT appropriate for the release (defaults to last committer). This should be `cumulus-from-npm` *except* in special cases such as incompatible backport branches.
```
Does this seem appropriate to resolve this concern?